frame/elections-phragmen/src/lib.rs

```diff
@@ -549,10 +549,13 @@ impl<T: Trait> Module<T> {
 			let members = members_with_stake.into_iter().map(|m| m.0).collect::<Vec<_>>();
 			let result = Ok(maybe_replacement.is_some());
 			let old = [who.clone()];
-			match maybe_replacement {
+			// We don't need to check the actual new members length because it will be the same
+			// as before or one less.
+			// TODO: Is this safe without ensure_can_change_members?
```

Comment:
Sounds fair to me

Reply:
This todo should be removed, or put into an issue.

Reply:
Yes. Sorry this isn't more clear. The Todos are implict questions to the reviewers :grimacing: